src/fswAlgorithms/orbitControl/lambertSolver/lambertSolver.cpp

```diff
@@ -173,6 +173,37 @@ double LambertSolver::x2tof(double x, int N, double lam)
     return tof;
 }
 
+/*! This method computes the derivatives of the time of flight curve T(x)
+    @param x free variable of Lambert's problem that satisfies the given time of flight
+    @param T requested non-dimensional time-of-flight
+    @param lam lambda parameter that defines the problem geometry
+    @return std::vector<double>
+*/
+std::vector<double> LambertSolver::dTdx(double x, double T, double lam)
+{
+    double DT; // dT/dx
+    double D2T; // d2T/dx2
+    double D3T; // d3T/dx3
+    if (abs(x - 1.0) < 1e-8){
+        // exact derivative for parabolic case
+
+        DT = 2.0/5.0*(pow(lam,5) - 1.0);
+        D2T = 0.0;
+        D3T = 0.0;
+    }
+    else{
+        double u = 1.0 - pow(x, 2);
+        double y = sqrt(1.0- pow(lam, 2) * u);
+        DT = 1.0/u*(3.0*T*x - 2.0 + 2.0 * pow(lam, 3) * x / y);
+        D2T = 1.0/u*(3.0*T + 5.0*x*DT + 2.0 * (1. - pow(lam, 2)) * pow(lam, 3) / pow(y, 3));
```

Comment:
:large_orange_diamond: Is there a neat place to add a line break into these two expressions such that they don't go over the desirable 120 character line length limit? 

I know I'm annoying :) and when we add in the automatic linting to the project these things will be checked by the build system and you'll be alerted. Until then, I'm here to bug you.

Reply:
Done.